tests/system/settings_test.go

```diff
@@ -88,6 +88,21 @@ func TestSettingsUpdate_noPermissions(t *testing.T) {
 		End()
 }
 
+func TestSettingsUpdate_validation(t *testing.T) {
+	h := newHelper(t)
+	helpers.AllowMe(h, types.ComponentRbacResource(), "settings.manage")
+	helpers.AllowMe(h, types.ComponentRbacResource(), "settings.read")
+
+	h.apiInit().
```

Comment:
are there other cases and input variants we could test here?

Reply:
Yeah, we can I will update it.